src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -433,6 +433,18 @@ private void OnUpdateGraphCompleted(AsyncTask task)
             var updateTask = (UpdateGraphAsyncTask)task;
             var messages = new Dictionary<Guid, string>();
 
+            // Mark all modified nodes as being updated.
+            // 
+            // In addition to marking modified nodes as being updated, their 
+            // warning states are cleared (which include the tool-tip).
+            foreach (var modifiedNode in updateTask.ModifiedNodes)
```

Comment:
@ke-yu in your PR description you said that this code needs to be moved to UI thread. Not sure if this change will help coz this code is still called from scheduler thread, is it not?